### DISCUSSION TITLE
[Feature] Allow custom head cell render function

### DISCUSSION THREAD

**Discussion by BennyAlex** [KevinVandy/material-react-table#1185] [Ideas]
Hello,

It would be nice to pass a custom render function for head cells, similar to the Cell prop on a column, just for all header of the table.
Would also help with adding tooltips etc.

**Comment by KevinVandy** [comment]
That has always already existed forever. See the docs
https://www.material-react-table.com/docs/guides/data-columns#custom-header-render

**Reply by BennyAlex** [reply]
@KevinVandy 
But thats for individual columns only.
It would much better to also have a a general function on the table itself used for all columns :D

**Reply by KevinVandy** [reply]
Use the defaultColumn table option. That applies defaults for all column defs
